Formula/hof.rb

```diff
@@ -26,6 +26,7 @@ class Hof < Formula
   end
 
   depends_on "go" => :build
+  depends_on "cue"
```

Comment:
Does it depend on `cue`, btw? It's not used by upstream's own formula either: https://github.com/hofstadter-io/homebrew-tap/blob/0feb45604ecd4bdc172dc50cdfc267b12f738578/hof.rb

Reply:
it does have cue inference in their code.

Reply:
For now, let me just get rid of this dependency, as it is easy to ship the formula.

Reply:
this is the code ref, otherwise they are vendoring it, https://github.com/hofstadter-io/hof/blob/0d88129256d3030a5578ae4e1335a20f7ac2a2c5/next/ci/dagger.mts#L267-L269

Reply:
@chenrui333, thanks for doing all this to get the update through!

removing cue should have no impact

context: While we don't require `cue` directly, we are so tightly coupled that we put it in as a dependency, because you're going to want it around. Users will know anyway, many come from CUE.

One thing I would like to do is get our own tap in sync with this one...